src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenDeconstructTests.cs

```diff
@@ -2607,9 +2607,15 @@ static void Main()
 
             var comp = CreateCompilationWithMscorlib(source, references: new[] { ValueTupleRef, SystemRuntimeFacadeRef });
             comp.VerifyDiagnostics(
-                // (6,14): error CS8134: Deconstruction must contain at least two variables.
+                // (6,20): error CS0103: The name 'x' does not exist in the current context
                 //         for ((var (x, y)) = Pair.Create(1, 2); ;) { }
-                Diagnostic(ErrorCode.ERR_DeconstructTooFewElements, "(var (x, y)) = Pair.Create(1, 2)").WithLocation(6, 14)
+                Diagnostic(ErrorCode.ERR_NameNotInContext, "x").WithArguments("x").WithLocation(6, 20),
```

Comment:
That's because it isn't a deconstruction. It was a bug in the old parser. This is an assignment to the result of calling a ref-returning method `var` on two arguments.

---
In reply to: [73446591](https://github.com/dotnet/roslyn/pull/12917#discussion_r73446591) [](ancestors = 73446591)